ALiAS_FAQs.md

```diff
@@ -0,0 +1,46 @@
+                                                FAQs
+Q1. What is ASET AliAS?
+Ans. ALiAS is not your regular Amity Club. ALiAS is a community, founded in 2010, which was developed with the aim to promote the use of Linux in the campus and get students familiar with the world of Open Source. The community members help students with their doubts pertaining to tech, conduct workshops, webinars,organize meet-ups (online and in-person) and most important get you connected.
+
+Q2. Why should I join ALiAS?
+Ans. ALiAS is by far, the largest open-source community in Amity University. We are
+open to developers not only inside the campus, but also from various places. We have got assistance
+from alumni and people from tech, near and far. We have got a presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.
+
+Q3. How is the community of AliAS?
+And. We have the most active developers in the campus. You will find people from nearly all domains,
+Web Development, Android Development, Artificial Intelligence, backend, frontend, industry people, designers, technical writers, theoretical CS enthusiasts,
+etc. And best of all, we have got a great reach for people and development.
+
+Q4. How should I join AliAS?
+Ans. Simple. Go and register yourself on http://asetalias.in/ . One of us will add you to the WhatsApp and Telegram groups.
+
+Q5. Having joined, where should I start?
+And. We request you to drop a message, stating your name, course duration, and your skills. Also
+let us know which field you are interested in, and we will guide you thereafter. Have one plan at a time.
+
+Q6. Best OS?
+Ans. Are you a designer? - Windows; Are you a developer? - Linux(Any)
+Windows has great graphics. It spoon feeds you everything. Many design softwares, games, drivers and
+support is easily available in windows. Installing anything is a cup of cake for anyone. But if you want
+to delve into softwares deeper, please forget your games and ease. Being open source community, we
+recommend installing Linux as dual boot, and get into ease with it. The only difference you will see in
+different distros is the GUI. Some software packages and libraries are pre-installed in some distros,
```

Comment:
Way more differences then GUI my man. 

And GUIs don't come with the distros. They come with desktop managers and window managers.